server/honeybadger/plugins.py

```diff
@@ -10,13 +10,11 @@ def get_coords_from_google(aps):
         data['wifiAccessPoints'].append(ap.serialized_for_google)
     data_json = json.dumps(data)
     headers = {'Content-Type': 'application/json'}
-    request = urllib2.Request(url, data=data_json, headers=headers)
-    response = urllib2.urlopen(request)
-    content = response.read()
-    logger.info("Google API response: {}".format(content))
+    request = requests.post(url=url, data=data_json, headers=headers)
+    logger.info("Google API response: {}".format(request.content))
     jsondata = None
     try:
-        jsondata = json.loads(content)
+        jsondata = json.loads(request.content)
```

Comment:
So, you don't need to load json here. If the response is json, then the response object will return it using the json method: `response.json()`. Make the change and give it a shot just to make sure.

Reply:
I'll give it a shot.

Reply:
The request.json() function works. The whole thing seems to work, but I will test it more on a proper environment. My python environment at home is messed up.